### PR TITLE
Add manifests to .All MetaPackage

### PR DESCRIPTION
**Pull request by JunTaoLuo**

### CODE REVIEW

**Comment by JunTaoLuo** [comment]
Not a big fan of adding 7zip into this repo but I need that to crack open the nupkg and insert more files. The alternative is to do this in Coherence-Signed instead.

File: build/repo.targets [hunk 1]
```diff
@@ -28,35 +35,49 @@
...
+
+    <ItemGroup>
+      <MetaPackageNupkg Include="$(COHERENCE_DROP_LOCATION)\ship\Microsoft.AspNetCore.All.*.nupkg" />
+      <ManifestFiles Include="$(ArtifactsDir)*.xml"/>
+    </ItemGroup>
+
+    <Copy SourceFiles="@(MetaPackageNupkg)" DestinationFolder="$(ArtifactsDir)" />
+    <Exec Command="$(ToolsDir)7za.exe x -y -scsUTF-8 -o@(MetaPackageNupkg->'$(TempDir)%(Filename)') %(MetaPackageNupkg.Identity) > nul" />
+    <Copy SourceFiles="@(ManifestFiles)" DestinationFolder="@(MetaPackageNupkg->'$(TempDir)%(Filename)\content\build\')" />
+    <Exec Command="$(ToolsDir)7za.exe a -tzip -mx9 -r -y %(MetaPackageNupkg.FullPath) * > nul" WorkingDirectory="@(MetaPackageNupkg->'$(TempDir)%(Filename)')" />
```
Comment (JunTaoLuo) [inline build/repo.targets]: Why aren't we using this in Coherence-Signed?

Comment (pranavkm) [inline build/repo.targets]: It was created before we had an sdk

**Comment by JunTaoLuo** [comment]
oops forgot to squash...

**Pull request merged**